doc/Comments.xml

```diff
@@ -139,6 +139,7 @@ text in your documentation, examples, mathematical chapters, etc..
 <Subsection Label="@Chapter">
 <Index Key="@Chapter"><C>@Chapter</C></Index>
 <Index Key="@ChapterLabel"><C>@ChapterLabel</C></Index>
+<Index Key="@ChapterTitle"><C>@ChapterTitle</C></Index>
 <Heading>@Chapter <A>name</A></Heading>
 Sets a chapter, all functions without separate info will be added to this chapter.
```

Comment:
While you are at it, perhaps also improve this text. How about: "Sets the active chapter. All subsequent functions which do not have an explicit chapter declared in their AutoDoc comment via [... err, how do we do it? Also via @Chapter?]  will be added to this chapter."

Reply:
via `@ChapterInfo`. That's a good idea.